packages/web-component/README.md

```diff
@@ -41,9 +41,11 @@ render(){
 - Now you can add the custom element to your HTML
 
 ```html
-<descope-wc project-id="<project-id>" flow-id="<flow-id>"></descope-wc>
+<descope-wc project-id="<project-id>" flow-id="<flow-id>" form='{ "email": "predefinedname@domain.com", "myCustomInput": "12" }' client="+JSON.stringify({ browserName: window.navigator.appName, browserVersion: window.navigator.appVersion })+"></descope-wc>
```

Comment:
you have an extrra `+` there which i think is redundant

also why do you use string notation in `form`  and json notation in `client`?


FYI I would give a smipler examples for `client

Reply:
1. fixed
2. I just copied this from the guide, can fix accordingly 
3. I will use only one key instead of 2